localisation/00_wc_job_titles.csv

```diff
@@ -63,3 +38,318 @@ job_spiritual_void_group;Shadow Priest;Pr
+job_chancellor_satyr_group;Viletongue;;;;;;;;;;;;;x
+job_chancellor_female_satyr_group;Viletongue;;;;;;;;;;;;;x
+job_marshal_satyr_group;Hellcaller;;;;;;;;;;;;;x
+job_marshal_female_satyr_group;Hellcaller;;;;;;;;;;;;;x
+job_treasurer_satyr_group;Wildshaper;;;;;;;;;;;;;x
+job_treasurer_female_satyr_group;Wildshaper;;;;;;;;;;;;;x
+job_spymaster_satyr_group;Shadowdancer;;;;;;;;;;;;;x
+job_spymaster_female_satyr_group;Shadowdancer;;;;;;;;;;;;;x
+job_chancellor_sethrak_group;Vizier;;;;;;;;;;;;;x
+job_chancellor_female_sethrak_group;Vizier;;;;;;;;;;;;;x
+job_marshal_sethrak_group;Warbringer;;;;;;;;;;;;;x
+job_marshal_female_sethrak_group;Warbringer;;;;;;;;;;;;;x
+job_treasurer_sethrak_group;Dunemaster;;;;;;;;;;;;;x
+job_treasurer_female_sethrak_group;Dunemistress;;;;;;;;;;;;;x
+job_spymaster_sethrak_group;Shadow;;;;;;;;;;;;;x
+job_spymaster_female_sethrak_group;Shadow;;;;;;;;;;;;;x
+job_chancellor_shathyar_group;Halahs'uhnish;;;;;;;;;;;;;x
+job_chancellor_female_shathyar_group;Halahs'uhnish;;;;;;;;;;;;;x
+job_marshal_shathyar_group;Phquathi'agthu;;;;;;;;;;;;;x
+job_marshal_female_shathyar_group;Phquathi'agthu;;;;;;;;;;;;;x
+job_treasurer_shathyar_group;Vorzz'uull;;;;;;;;;;;;;x
+job_treasurer_female_shathyar_group;Vorzz'uull;;;;;;;;;;;;;x
+job_spymaster_shathyar_group;Gaz'qor'magg;;;;;;;;;;;;;x
+job_spymaster_female_sethrak_group;Gaz'qor'magg;;;;;;;;;;;;;x
+job_chancellor_tauren_group;Spiritwalker;;;;;;;;;;;;;x
+job_chancellor_female_tauren_group;Spiritwalker;;;;;;;;;;;;;x
+job_marshal_tauren_group;Brave;;;;;;;;;;;;;x
+job_marshal_female_tauren_group;Brave;;;;;;;;;;;;;x
+job_treasurer_tauren_group;Master;;;;;;;;;;;;;x
+job_treasurer_female_tauren_group;Master;;;;;;;;;;;;;x
+job_spymaster_tauren_group;Chief Scout;;;;;;;;;;;;;x
+job_spymaster_female_tauren_group;Chief Scout;;;;;;;;;;;;;x
+job_chancellor_titanforged_group;Keeper;;;;;;;;;;;;;x
+job_chancellor_female_titanforged_group;Keeper;;;;;;;;;;;;;x
+job_marshal_titanforged_group;Protector;;;;;;;;;;;;;x
+job_marshal_female_titanforged_group;Protector;;;;;;;;;;;;;x
+job_treasurer_titanforged_group;Architect;;;;;;;;;;;;;x
+job_treasurer_female_titanforged_group;Architect;;;;;;;;;;;;;x
+job_spymaster_titanforged_group;Seeker;;;;;;;;;;;;;x
+job_spymaster_female_titanforged_group;Seeker;;;;;;;;;;;;;x
+job_chancellor_trogg_group;Boss;;;;;;;;;;;;;x
+job_chancellor_female_trogg_group;Boss;;;;;;;;;;;;;x
+job_marshal_trogg_group;Warrior;;;;;;;;;;;;;x
+job_marshal_female_trogg_group;Warrior;;;;;;;;;;;;;x
+job_treasurer_trogg_group;Drudge;;;;;;;;;;;;;x
+job_treasurer_female_trogg_group;Drudge;;;;;;;;;;;;;x
+job_spymaster_trogg_group;Scout;;;;;;;;;;;;;x
+job_spymaster_female_trogg_group;Scout;;;;;;;;;;;;;x
+job_chancellor_troll_group;Overseer;;;;;;;;;;;;;x
+job_chancellor_female_troll_group;Overseer;;;;;;;;;;;;;x
+job_marshal_troll_group;Warlord;;;;;;;;;;;;;x
+job_marshal_female_troll_group;Warlord;;;;;;;;;;;;;x
+job_treasurer_troll_group;Elder;;;;;;;;;;;;;x
+job_treasurer_female_troll_group;Elder;;;;;;;;;;;;;x
```

Comment:
Identical to vanilla title. I suggest **_Treasurer_** or **_Steward_** though it's not lore reference.